toolkit/quickrebuild_autoconfig.mk

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Please consider adding some very basic documentation on the new options in the docs folder

Reply:
I think Daniel's addressing that separately in #5218.

Reply:
I've moved the docs here now